src/V3Options.h

```diff
@@ -533,6 +534,9 @@ class V3Options final {
     VTimescale timeComputePrec(const VTimescale& flag) const;
     VTimescale timeComputeUnit(const VTimescale& flag) const;
     int traceDepth() const { return m_traceDepth; }
+
+    int publicDepth() const { return m_publicDepth; }
+
```

Comment:
```suggestion
    int publicDepth() const { return m_publicDepth; }
```
And put in sorted spot.
And sort